### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by NightNutSky**
Although you explained something about it, but I'm sorry if I did something wrong. Let me know what to fix, if any. I will check this PR in about 6-8 hours.

### CODE REVIEW

**Comment by NightNutSky** [comment]
Just an FYI, I named the folder as "values-ru" because... 
Screenshot_20220913-070706~2

File: app/src/main/res/values-ru/translation.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    <!-- Edit your name here according to translated language -->
+    <string name="english_contributors">Vaibhav</string>
+    <string name="russian_contributors">NightNutSky</string>
+
+
+
+    <!-- Don't Edit this -->
+    <string-array name="contributors">
+        <item>@string/english_contributors</item>
```
Comment (dead8309) [inline app/src/main/res/values-ru/translation.xml:18]:
```xml
<item>@string/russian_contributors</item>
```
Should be added here

**Review by dead8309 — CHANGES_REQUESTED** [review]
Change the directory name to `values-ba`

File: app/src/main/res/values-ba/translation.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (dead8309) [inline app/src/main/res/values-ba/translation.xml:1]: Remove this file and put the fields in `strings.xml` located in `app/src/main/res/values`

**Comment by dead8309** [comment]
Thanks for your contribution.

**Pull request merged**